src/pages/events/createEventsPageComponenets/CreateEvetsStepper.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import { StepIconProps } from '@mui/material/StepIcon';
+import ColorlibStepIconRoot from "../createEvents/stepper/ColorlibStepIconRoot";
+import useCreateEventStyles from "../../../assets/styleJs/events/createEvent";
+import ColorlibConnector from "../createEvents/stepper/ColorlibConnector";
+
+const ColorlibStepIcon=({active, className, completed, icon}: StepIconProps) =>{
+
+  const icons: { [index: string]: string } = {
+    1: '1',
+    2: '2',
+    3: '3',
+  };
```

Comment:
?

Reply:
it gives MUI as styled component, it's for making stepper(each values is for stepper's each step rendering)

Reply:
should keys be also 1 2 3?

Reply:
yes, there can be also words